src/actions/channels.js

```diff
@@ -128,7 +132,7 @@ export function createDirectChannel(userId, otherUserId) {
             },
             {
                 type: PreferenceTypes.RECEIVED_PREFERENCES,
-                data: [{category: Preferences.CATEGORY_DIRECT_CHANNEL_SHOW, name: otherUserId, value: 'true'}]
+                data: preference
```

Comment:
can we name this `preferences` as is an array and not just the preference object?